Makefile

```diff
@@ -89,36 +89,48 @@ ifeq ($(TARGET_ARCH),x64)
 	TARGET_ARCH_DEBIAN = amd64
 endif
 
-TEMPORARY_DIRECTORY = release/.tmp
```

Comment:
IMHO there's no reason not to keep referring to this as `$(TEMPORARY_DIRECTORY)` throughout the rest of the `Makefile`?